2.cellprofiler_processing/CellProfiler_input/CFReT_project_CL.cppipe

```diff
@@ -173,10 +173,10 @@ MeasureGranularity:[module_num:9|svn_version:'Unknown'|variable_revision_number:
 
 MeasureObjectIntensity:[module_num:10|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
     Select images to measure:Actin, ER, Golgi, Hoechst, Mitochondria
-    Select objects to measure:Cells, Cytoplasm, Nuclei, OrigNuclei
+    Select objects to measure:Cells, Cytoplasm, Nuclei
 
 MeasureImageIntensity:[module_num:11|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
-    Select images to measure:Actin, ER, Golgi, Hoechst
+    Select images to measure:Actin, ER, Golgi, Hoechst, Mitochondria
```

Comment:
nice! It seems you caught this bug too?

Reply:
Ahh darn my bad. And yes I did catch it! I do notice sometimes when using the CellProfiler GUI that is will randomly unselect things, which was something I discovered in the NF1 pipelines.

Let me convert it now!